scripts/add_global_subscriptions.py

```diff
@@ -0,0 +1,44 @@
+"""
+This migration subscribes each user to USER_SUBSCRIPTIONS_AVAILABLE if a subscription
+does not already exist.
+"""
+
+import logging
+import sys
+
+from website.app import init_app
+from website import models
+from website.notifications.model import NotificationSubscription
+from website.notifications import constants
+from website.notifications.utils import to_subscription_key
+
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+
+app = init_app()
+
+
+def add_global_subscriptions():
+
+    notification_type = 'email_transactional'
+    user_events = constants.USER_SUBSCRIPTIONS_AVAILABLE
+
+    for user in models.User.find():
```

Comment:
We'll probably want to exclude inactive users (i.e. any users that don't meet the criteria for the `User.is_active` property).